package.json

```diff
@@ -3,6 +3,7 @@
   "version": "2.0.6",
   "author": "Jeff Dickey @dickeyxxxx",
   "dependencies": {
+
```

Comment:
BTW, do we have any documentation for a checklist of things to do before submitting a PR? Like "run fixpack, use xyz formatter", etc.